packages/astro/src/@types/astro.ts

```diff
@@ -1497,66 +1497,68 @@ export interface AstroUserConfig {
 		 *
 		 * Controls the routing strategy to determine your site URLs. Set this based on your folder/URL path configuration for your default language.
 		 */
-		routing?: {
-			/**
-			 * @docs
-			 * @name i18n.routing.prefixDefaultLocale
-			 * @kind h4
-			 * @type {boolean}
-			 * @default `false`
-			 * @version 3.7.0
-			 * @description
-			 *
-			 * When `false`, only non-default languages will display a language prefix.
-			 * The `defaultLocale` will not show a language prefix and content files do not exist in a localized folder.
-			 *  URLs will be of the form `example.com/[locale]/content/` for all non-default languages, but `example.com/content/` for the default locale.
-			 *
-			 * When `true`, all URLs will display a language prefix.
-			 * URLs will be of the form `example.com/[locale]/content/` for every route, including the default language.
-			 * Localized folders are used for every language, including the default.
-			 */
-			prefixDefaultLocale?: boolean;
-
-			/**
-			 * @docs
-			 * @name i18n.routing.redirectToDefaultLocale
-			 * @kind h4
-			 * @type {boolean}
-			 * @default `true`
-			 * @version 4.2.0
-			 * @description
-			 *
-			 * Configures whether or not the home URL (`/`) generated by `src/pages/index.astro`
-			 * will redirect to `/[defaultLocale]` when `prefixDefaultLocale: true` is set.
-			 *
-			 * Set `redirectToDefaultLocale: false` to disable this automatic redirection at the root of your site:
-			 * ```js
-			 * // astro.config.mjs
-			 * export default defineConfig({
-			 *   i18n:{
-			 *     defaultLocale: "en",
-			 * 		locales: ["en", "fr"],
-			 *     routing: {
-			 *       prefixDefaultLocale: true,
-			 *       redirectToDefaultLocale: false
-			 *     }
-			 *   }
-			 * })
-			 *```
-			 * */
-			redirectToDefaultLocale?: boolean;
-
-			/**
-			 * @name i18n.routing.strategy
-			 * @type {"pathname"}
-			 * @default `"pathname"`
-			 * @version 3.7.0
-			 * @description
-			 *
-			 * - `"pathname": The strategy is applied to the pathname of the URLs
-			 */
-			strategy?: 'pathname';
-		};
+		routing?:
+			| 'manual'
```

Comment:
Just checking, are we waiting for a docs entry for "manual" here? This should have its own entry that shows up in the docs config reference file.

For reference, this is the text from the docs guide page, to help craft something:

> When this option is enabled, Astro will **disable** its i18n middleware so that you can implement your own custom logic. No other `routing` options (e.g. `prefixDefaultLocale`) may be configured with `routing: "manual"`.
> 
> You will be responsible for writing your own routing logic, or executing Astro's i18n middleware manually alongside your own.

Reply:
That's done